plugin/PDFDocumentPage.qml

```diff
@@ -206,6 +211,26 @@ DocumentPage {
         }
     }
 
+    PDFContextMenu {
+        id: contextMenuLinks
+        //% "External link"
+        title: qsTrId("sailfish-office-tl-pdf-link")
+        MenuItem {
+            text: (contextMenuLinks.message.indexOf("http:") === 0 ||
+            contextMenuLinks.message.indexOf("https:") === 0) ?
+            //% "Open in browser"
+            qsTrId("sailfish-office-me-pdf-open-link") :
+            //% "Open in external application"
+            qsTrId("sailfish-office-me-pdf-open-external")
```

Comment:
Please have some indentation on these. Starting all lines at same position makes it harder to notice where property value starts. Operators preferably at start of lines: https://wiki.qt.io/Qt_Coding_Style#Line_breaks